doc/build/orm/inheritance.rst

```diff
@@ -3,11 +3,15 @@
 Mapping Class Inheritance Hierarchies
 =====================================
 
-SQLAlchemy supports three forms of inheritance: **single table inheritance**,
-where several types of classes are represented by a single table, **concrete
-table inheritance**, where each type of class is represented by independent
-tables, and **joined table inheritance**, where the class hierarchy is broken
-up among dependent tables, each class represented by its own table that only
+SQLAlchemy supports three forms of inheritance:
+
+* **single table inheritance** – several types of classes are represented by a single table;
+
+* **concrete table inheritance** – each type of class is represented by independent tables;
+
+* **joined table inheritance** – the class hierarchy is broken up among dependent tables.
+
+Each class represented by its own table that only
```

Comment:
I think this part and the phrase below refers to the "joined table inheritance" section